css/global.css

```diff
@@ -79,6 +196,11 @@ Wrappers (container, flexbox, etc.)
 /* Container */
 
 /* Wrappers */
+.wrapper {
```

Comment:
Really useful for setting up global classes! we will probably add more in the future!